RetailCoder.VBE/Settings/ConfigurationLoader.cs

```diff
@@ -32,16 +32,18 @@ public class ConfigurationLoader : IGeneralConfigService
         private readonly IConfigProvider<CodeInspectionSettings> _inspectionProvider;
         private readonly IConfigProvider<UnitTestSettings> _unitTestProvider;
         private readonly IConfigProvider<IndenterSettings> _indenterProvider;
+        private readonly IConfigProvider<WindowSettings> _windowProvider;
 
         public ConfigurationLoader(IConfigProvider<GeneralSettings> generalProvider, IConfigProvider<HotkeySettings> hotkeyProvider, IConfigProvider<ToDoListSettings> todoProvider,
-                                   IConfigProvider<CodeInspectionSettings> inspectionProvider, IConfigProvider<UnitTestSettings> unitTestProvider, IConfigProvider<IndenterSettings> indenterProvider)
+                                   IConfigProvider<CodeInspectionSettings> inspectionProvider, IConfigProvider<UnitTestSettings> unitTestProvider, IConfigProvider<IndenterSettings> indenterProvider, IConfigProvider<WindowSettings> windowProvider)
         {
```

Comment:
This is the signature change that the tests aren't liking for a reason we need to find out before we can merge.